docs/sources/datasources/jaeger.md

```diff
@@ -56,6 +56,60 @@ To perform a search, set the query type selector to Search, then use the followi
 - Max Duration - Filter all traces with a duration lower than the set value. Possible values are `1.2s, 100ms, 500us`.
 - Limit - Limits the number of traces returned.
 
+## Upload JSON trace file
+
+{{< figure src="/static/img/docs/explore/jaeger-upload-json.png" class="docs-image--no-shadow" caption="Screenshot of the Jaeger data source in explore with upload selected" >}}
+
+You can upload JSON file that contains a single trace to visualize it. If the JSON contains multiple traces then the first one is going to be used. The JSON schema needs to look like this:
```

Comment:
Could we link to the jaeger doc to be clear this is something you can download from jaeger-ui?

Reply:
`The JSON schema needs to look like this`

Also as this is not a json schema but an example of the json I would probably reword it a bit.

Reply:
> Could we link to the jaeger doc to be clear this is something you can download from jaeger-ui?

I couldn't find anything that can be linked for this.